script.js

```diff
@@ -0,0 +1,115 @@
+const spinnerLoad = document.getElementById("sppinner");
+const inputField = document.querySelector("#sequenceInput");
+const errorDivPlaceholder = document.querySelector("#error-div-placeholder");
+const calcDiv = document.querySelector("#calc-div");
+const btn = document.querySelector("#resultBtn");
+const errorDivOver50 = document.createElement("div");
+errorDivOver50.innerHTML = "Can't be larger than 50";
+errorDivOver50.setAttribute("id", "error50");
+errorDivOver50.classList.add(
+  "bg-danger",
+  "border-danger",
+  "text-danger",
+  "bg-opacity-25",
+  "rounded-1",
+  "error-box50"
+);
+
+const checkBox = document.getElementById("checkbox");
+function fibcalc(sequence) {
+  let url = "http://localhost:5050/fibonacci/" + sequence;
+  fetch(url)
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      } 
+      throw new Error("Server Error: 42 is the meaning of life"); 
+    }) 
+    .then((data) => {
+      calcList();
+      document.getElementById("resultNum").innerHTML = data["result"];
+    })
+    .catch((error) => {
+      cleanUp();
+      let errorDiv42 = document.createElement("div");
+      errorDiv42.innerHTML = error.message;
+      errorDiv42.classList.add("text-danger", "ms-1", "text-opacity-75");
+      errorDiv42.setAttribute("id", "error42");
+      calcDiv.appendChild(errorDiv42);
+    });
+}
+
+function cleanUp() {
```

Comment:
You should have fix this function or at least talk to Yonatan about it, please ask us about me